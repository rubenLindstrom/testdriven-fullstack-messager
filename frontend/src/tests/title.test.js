import React from "react";
import Title from "../components/title";

import Enzyme, { mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe("Title", () => {
  it("renders without crashing", () => {
    const component = mount(<Title />);
    expect(component).toMatchSnapshot();
  });
});
