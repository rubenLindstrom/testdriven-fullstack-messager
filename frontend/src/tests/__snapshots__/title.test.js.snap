// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders without crashing 1`] = `ReactWrapper {}`;
